packages/time-utilities/src/lib/constants.ts

```diff
@@ -1,3 +1,6 @@
+/* eslint-disable @typescript-eslint/naming-convention */
+import { DurationFormatAssetsTime, TimeTypes } from '..';
```

Comment:
Can you change this to the file(s) where these imports actually are? I'm not much a fan of importing from `src/index.ts` in other files.

Reply:
Resolved in 38c91ba